pkg/syncthing/process.go

```diff
@@ -16,22 +14,47 @@
 package syncthing
 
 import (
-	"os"
-	"strings"
+	"time"
+
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/shirou/gopsutil/process"
 )
 
-func terminate(pid int, wait bool) error {
-	proc := os.Process{Pid: pid}
-	if err := proc.Signal(os.Interrupt); err != nil {
-		if strings.Contains(err.Error(), "process already finished") {
-			return nil
-		}
+func getParentExe(p *process.Process) string {
```

Comment:
if this just for windows? If this code depends on windows components, you need to put it in the  process_windows.go folder, so it doesn't get compiled on Linux or macos (it's based on build tag at the top of the file)

Reply:
Is not clear what 'Exe' means in this context, maybe rename to something more explicit? (I believe this is returning the path to the parent of `p` ?